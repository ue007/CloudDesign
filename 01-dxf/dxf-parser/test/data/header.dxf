  0
SECTION
  2
HEADER
  9
$ACADVER
  1
AC1027
  9
$ACADMAINTVER
 70
    55
  9
$DWGCODEPAGE
  3
ANSI_1252
  9
$LASTSAVEDBY
  1
Shane
  9
$REQUIREDVERSIONS
160
                 0
  9
$INSBASE
 10
0.0
 20
0.0
 30
0.0
  9
$EXTMIN
 10
146.8403717847312
 20
141.8294137695975
 30
-5.934454319554902
  9
$EXTMAX
 10
762.6679804989191
 20
545.821967323924
 30
6.065545680445099
  9
$LIMMIN
 10
0.0
 20
0.0
  9
$LIMMAX
 10
1000.0
 20
800.0
  9
$ORTHOMODE
 70
     1
  9
$REGENMODE
 70
     1
  9
$FILLMODE
 70
     1
  9
$QTEXTMODE
 70
     0
  9
$MIRRTEXT
 70
     0
  9
$LTSCALE
 40
1.0
  9
$ATTMODE
 70
     1
  9
$TEXTSIZE
 40
0.1875
  9
$TRACEWID
 40
0.05
  9
$TEXTSTYLE
  7
FG-Title
  9
$CLAYER
  8
FG-Dtl-Hidden
  9
$CELTYPE
  6
ByLayer
  9
$CECOLOR
 62
   256
  9
$CELTSCALE
 40
1.0
  9
$DISPSILH
 70
     1
  9
$DIMSCALE
 40
48.0
  9
$DIMASZ
 40
0.125
  9
$DIMEXO
 40
0.0625
  9
$DIMDLI
 40
0.375
  9
$DIMRND
 40
0.0
  9
$DIMDLE
 40
0.0
  9
$DIMEXE
 40
0.0625
  9
$DIMTP
 40
0.0
  9
$DIMTM
 40
0.0
  9
$DIMTXT
 40
0.09375
  9
$DIMCEN
 40
0.09
  9
$DIMTSZ
 40
0.0
  9
$DIMTOL
 70
     0
  9
$DIMLIM
 70
     0
  9
$DIMTIH
 70
     0
  9
$DIMTOH
 70
     0
  9
$DIMSE1
 70
     0
  9
$DIMSE2
 70
     0
  9
$DIMTAD
 70
     1
  9
$DIMZIN
 70
     0
  9
$DIMBLK
  1
Oblique
  9
$DIMASO
 70
     1
  9
$DIMSHO
 70
     1
  9
$DIMPOST
  1
"
  9
$DIMAPOST
  1

  9
$DIMALT
 70
     0
  9
$DIMALTD
 70
     4
  9
$DIMALTF
 40
1.0
  9
$DIMLFAC
 40
1.0
  9
$DIMTOFL
 70
     1
  9
$DIMTVP
 40
0.0
  9
$DIMTIX
 70
     0
  9
$DIMSOXD
 70
     0
  9
$DIMSAH
 70
     0
  9
$DIMBLK1
  1

  9
$DIMBLK2
  1

  9
$DIMSTYLE
  2
STANDARD
  9
$DIMCLRD
 70
   256
  9
$DIMCLRE
 70
   256
  9
$DIMCLRT
 70
   256
  9
$DIMTFAC
 40
1.0
  9
$DIMGAP
 40
0.03125
  9
$DIMJUST
 70
     0
  9
$DIMSD1
 70
     0
  9
$DIMSD2
 70
     0
  9
$DIMTOLJ
 70
     1
  9
$DIMTZIN
 70
     0
  9
$DIMALTZ
 70
     0
  9
$DIMALTTZ
 70
     0
  9
$DIMUPT
 70
     0
  9
$DIMDEC
 70
     4
  9
$DIMTDEC
 70
     4
  9
$DIMALTU
 70
     6
  9
$DIMALTTD
 70
     4
  9
$DIMTXSTY
  7
FG-Note
  9
$DIMAUNIT
 70
     0
  9
$DIMADEC
 70
     0
  9
$DIMALTRND
 40
0.0
  9
$DIMAZIN
 70
     0
  9
$DIMDSEP
 70
    46
  9
$DIMATFIT
 70
     0
  9
$DIMFRAC
 70
     1
  9
$DIMLDRBLK
  1

  9
$DIMLUNIT
 70
     5
  9
$DIMLWD
 70
    -1
  9
$DIMLWE
 70
    -1
  9
$DIMTMOVE
 70
     0
  9
$DIMFXL
 40
0.18
  9
$DIMFXLON
 70
     0
  9
$DIMJOGANG
 40
0.7853981633974483
  9
$DIMTFILL
 70
     0
  9
$DIMTFILLCLR
 70
     0
  9
$DIMARCSYM
 70
     0
  9
$DIMLTYPE
  6
ByLayer
  9
$DIMLTEX1
  6
ByLayer
  9
$DIMLTEX2
  6
ByLayer
  9
$DIMTXTDIRECTION
 70
     0
  9
$LUNITS
 70
     2
  9
$LUPREC
 70
     5
  9
$SKETCHINC
 40
0.1
  9
$FILLETRAD
 40
0.0
  9
$AUNITS
 70
     0
  9
$AUPREC
 70
     0
  9
$MENU
  1
.
  9
$ELEVATION
 40
0.0
  9
$PELEVATION
 40
0.0
  9
$THICKNESS
 40
0.0
  9
$LIMCHECK
 70
     0
  9
$CHAMFERA
 40
0.0
  9
$CHAMFERB
 40
0.0
  9
$CHAMFERC
 40
0.0
  9
$CHAMFERD
 40
0.0
  9
$SKPOLY
 70
     0
  9
$TDCREATE
 40
2455379.770598715
  9
$TDUCREATE
 40
2455379.770598715
  9
$TDUPDATE
 40
2455614.973897338
  9
$TDUUPDATE
 40
2455614.973897338
  9
$TDINDWG
 40
1.6838402431
  9
$TDUSRTIMER
 40
1.6838402431
  9
$USRTIMER
 70
     1
  9
$ANGBASE
 50
0.0
  9
$ANGDIR
 70
     0
  9
$PDMODE
 70
     0
  9
$PDSIZE
 40
0.0
  9
$PLINEWID
 40
0.0
  9
$SPLFRAME
 70
     0
  9
$SPLINETYPE
 70
     6
  9
$SPLINESEGS
 70
     8
  9
$HANDSEED
  5
BFA8
  9
$SURFTAB1
 70
     6
  9
$SURFTAB2
 70
     6
  9
$SURFTYPE
 70
     6
  9
$SURFU
 70
     6
  9
$SURFV
 70
     6
  9
$UCSBASE
  2

  9
$UCSNAME
  2

  9
$UCSORG
 10
0.0
 20
0.0
 30
0.0
  9
$UCSXDIR
 10
1.0
 20
0.0
 30
0.0
  9
$UCSYDIR
 10
0.0
 20
1.0
 30
0.0
  9
$UCSORTHOREF
  2

  9
$UCSORTHOVIEW
 70
     0
  9
$UCSORGTOP
 10
0.0
 20
0.0
 30
0.0
  9
$UCSORGBOTTOM
 10
0.0
 20
0.0
 30
0.0
  9
$UCSORGLEFT
 10
0.0
 20
0.0
 30
0.0
  9
$UCSORGRIGHT
 10
0.0
 20
0.0
 30
0.0
  9
$UCSORGFRONT
 10
0.0
 20
0.0
 30
0.0
  9
$UCSORGBACK
 10
0.0
 20
0.0
 30
0.0
  9
$PUCSBASE
  2

  9
$PUCSNAME
  2

  9
$PUCSORG
 10
0.0
 20
0.0
 30
0.0
  9
$PUCSXDIR
 10
1.0
 20
0.0
 30
0.0
  9
$PUCSYDIR
 10
0.0
 20
1.0
 30
0.0
  9
$PUCSORTHOREF
  2

  9
$PUCSORTHOVIEW
 70
     0
  9
$PUCSORGTOP
 10
0.0
 20
0.0
 30
0.0
  9
$PUCSORGBOTTOM
 10
0.0
 20
0.0
 30
0.0
  9
$PUCSORGLEFT
 10
0.0
 20
0.0
 30
0.0
  9
$PUCSORGRIGHT
 10
0.0
 20
0.0
 30
0.0
  9
$PUCSORGFRONT
 10
0.0
 20
0.0
 30
0.0
  9
$PUCSORGBACK
 10
0.0
 20
0.0
 30
0.0
  9
$USERI1
 70
     0
  9
$USERI2
 70
     0
  9
$USERI3
 70
     0
  9
$USERI4
 70
     0
  9
$USERI5
 70
     0
  9
$USERR1
 40
0.0
  9
$USERR2
 40
0.0
  9
$USERR3
 40
0.0
  9
$USERR4
 40
0.0
  9
$USERR5
 40
0.0
  9
$WORLDVIEW
 70
     1
  9
$SHADEDGE
 70
     3
  9
$SHADEDIF
 70
    70
  9
$TILEMODE
 70
     0
  9
$MAXACTVP
 70
    64
  9
$PINSBASE
 10
0.0
 20
0.0
 30
0.0
  9
$PLIMCHECK
 70
     0
  9
$PEXTMIN
 10
-0.25
 20
-0.1305130401371635
 30
0.0
  9
$PEXTMAX
 10
16.5
 20
10.61948695986283
 30
0.0
  9
$PLIMMIN
 10
-0.2499321702471884
 20
-0.3750450970290743
  9
$PLIMMAX
 10
16.75006734916103
 20
10.62495466267503
  9
$UNITMODE
 70
     0
  9
$VISRETAIN
 70
     1
  9
$PLINEGEN
 70
     0
  9
$PSLTSCALE
 70
     1
  9
$TREEDEPTH
 70
  3020
  9
$CMLSTYLE
  2
Standard
  9
$CMLJUST
 70
     0
  9
$CMLSCALE
 40
1.0
  9
$PROXYGRAPHICS
 70
     1
  9
$MEASUREMENT
 70
     0
  9
$CELWEIGHT
370
    -1
  9
$ENDCAPS
280
     0
  9
$JOINSTYLE
280
     0
  9
$LWDISPLAY
290
     0
  9
$INSUNITS
 70
     0
  9
$HYPERLINKBASE
  1

  9
$STYLESHEET
  1
Florida.ctb
  9
$XEDIT
290
     1
  9
$CEPSNTYPE
380
     0
  9
$PSTYLEMODE
290
     1
  9
$FINGERPRINTGUID
  2
{E451773B-08B1-4376-9DA5-8F5F0D6C19EA}
  9
$VERSIONGUID
  2
{6EFBAB5D-9128-4E51-B2D4-822E03762ACA}
  9
$EXTNAMES
290
     1
  9
$PSVPSCALE
 40
0.0
  9
$OLESTARTUP
290
     0
  9
$SORTENTS
280
   127
  9
$INDEXCTL
280
     0
  9
$HIDETEXT
280
     1
  9
$XCLIPFRAME
280
     0
  9
$HALOGAP
280
     0
  9
$OBSCOLOR
 70
   257
  9
$OBSLTYPE
280
     0
  9
$INTERSECTIONDISPLAY
280
     0
  9
$INTERSECTIONCOLOR
 70
   257
  9
$DIMASSOC
280
     2
  9
$PROJECTNAME
  1

  9
$CAMERADISPLAY
290
     0
  9
$LENSLENGTH
 40
50.0
  9
$CAMERAHEIGHT
 40
0.0
  9
$STEPSPERSEC
 40
2.0
  9
$STEPSIZE
 40
6.0
  9
$3DDWFPREC
 40
2.0
  9
$PSOLWIDTH
 40
0.25
  9
$PSOLHEIGHT
 40
4.0
  9
$LOFTANG1
 40
1.570796326794896
  9
$LOFTANG2
 40
1.570796326794896
  9
$LOFTMAG1
 40
0.0
  9
$LOFTMAG2
 40
0.0
  9
$LOFTPARAM
 70
     7
  9
$LOFTNORMALS
280
     1
  9
$LATITUDE
 40
37.795
  9
$LONGITUDE
 40
-122.394
  9
$NORTHDIRECTION
 40
0.0
  9
$TIMEZONE
 70
 -8000
  9
$LIGHTGLYPHDISPLAY
280
     1
  9
$TILEMODELIGHTSYNCH
280
     1
  9
$CMATERIAL
347
96
  9
$SOLIDHIST
280
     0
  9
$SHOWHIST
280
     1
  9
$DWFFRAME
280
     2
  9
$DGNFRAME
280
     0
  9
$REALWORLDSCALE
290
     1
  9
$INTERFERECOLOR
 62
     1
  9
$INTERFEREOBJVS
345
A3
  9
$INTERFEREVPVS
346
BF84
  9
$CSHADOW
280
     0
  9
$SHADOWPLANELOCATION
 40
0.0
  0
ENDSEC
  0
EOF