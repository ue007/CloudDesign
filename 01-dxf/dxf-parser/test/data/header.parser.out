{
    "header": {
        "$ACADVER": "AC1027",
        "$ACADMAINTVER": 55,
        "$DWGCODEPAGE": "ANSI_1252",
        "$LASTSAVEDBY": "Shane",
        "$REQUIREDVERSIONS": 0,
        "$INSBASE": {
            "x": 0,
            "y": 0,
            "z": 0
        },
        "$EXTMIN": {
            "x": 146.8403717847312,
            "y": 141.8294137695975,
            "z": -5.934454319554902
        },
        "$EXTMAX": {
            "x": 762.6679804989191,
            "y": 545.821967323924,
            "z": 6.065545680445099
        },
        "$LIMMIN": {
            "x": 0,
            "y": 0
        },
        "$LIMMAX": {
            "x": 1000,
            "y": 800
        },
        "$ORTHOMODE": 1,
        "$REGENMODE": 1,
        "$FILLMODE": 1,
        "$QTEXTMODE": 0,
        "$MIRRTEXT": 0,
        "$LTSCALE": 1,
        "$ATTMODE": 1,
        "$TEXTSIZE": 0.1875,
        "$TRACEWID": 0.05,
        "$TEXTSTYLE": "FG-Title",
        "$CLAYER": "FG-Dtl-Hidden",
        "$CELTYPE": "ByLayer",
        "$CECOLOR": 256,
        "$CELTSCALE": 1,
        "$DISPSILH": 1,
        "$DIMSCALE": 48,
        "$DIMASZ": 0.125,
        "$DIMEXO": 0.0625,
        "$DIMDLI": 0.375,
        "$DIMRND": 0,
        "$DIMDLE": 0,
        "$DIMEXE": 0.0625,
        "$DIMTP": 0,
        "$DIMTM": 0,
        "$DIMTXT": 0.09375,
        "$DIMCEN": 0.09,
        "$DIMTSZ": 0,
        "$DIMTOL": 0,
        "$DIMLIM": 0,
        "$DIMTIH": 0,
        "$DIMTOH": 0,
        "$DIMSE1": 0,
        "$DIMSE2": 0,
        "$DIMTAD": 1,
        "$DIMZIN": 0,
        "$DIMBLK": "Oblique",
        "$DIMASO": 1,
        "$DIMSHO": 1,
        "$DIMPOST": "\"",
        "$DIMAPOST": "",
        "$DIMALT": 0,
        "$DIMALTD": 4,
        "$DIMALTF": 1,
        "$DIMLFAC": 1,
        "$DIMTOFL": 1,
        "$DIMTVP": 0,
        "$DIMTIX": 0,
        "$DIMSOXD": 0,
        "$DIMSAH": 0,
        "$DIMBLK1": "",
        "$DIMBLK2": "",
        "$DIMSTYLE": "STANDARD",
        "$DIMCLRD": 256,
        "$DIMCLRE": 256,
        "$DIMCLRT": 256,
        "$DIMTFAC": 1,
        "$DIMGAP": 0.03125,
        "$DIMJUST": 0,
        "$DIMSD1": 0,
        "$DIMSD2": 0,
        "$DIMTOLJ": 1,
        "$DIMTZIN": 0,
        "$DIMALTZ": 0,
        "$DIMALTTZ": 0,
        "$DIMUPT": 0,
        "$DIMDEC": 4,
        "$DIMTDEC": 4,
        "$DIMALTU": 6,
        "$DIMALTTD": 4,
        "$DIMTXSTY": "FG-Note",
        "$DIMAUNIT": 0,
        "$DIMADEC": 0,
        "$DIMALTRND": 0,
        "$DIMAZIN": 0,
        "$DIMDSEP": 46,
        "$DIMATFIT": 0,
        "$DIMFRAC": 1,
        "$DIMLDRBLK": "",
        "$DIMLUNIT": 5,
        "$DIMLWD": -1,
        "$DIMLWE": -1,
        "$DIMTMOVE": 0,
        "$DIMFXL": 0.18,
        "$DIMFXLON": 0,
        "$DIMJOGANG": 0.7853981633974483,
        "$DIMTFILL": 0,
        "$DIMTFILLCLR": 0,
        "$DIMARCSYM": 0,
        "$DIMLTYPE": "ByLayer",
        "$DIMLTEX1": "ByLayer",
        "$DIMLTEX2": "ByLayer",
        "$DIMTXTDIRECTION": 0,
        "$LUNITS": 2,
        "$LUPREC": 5,
        "$SKETCHINC": 0.1,
        "$FILLETRAD": 0,
        "$AUNITS": 0,
        "$AUPREC": 0,
        "$MENU": ".",
        "$ELEVATION": 0,
        "$PELEVATION": 0,
        "$THICKNESS": 0,
        "$LIMCHECK": 0,
        "$CHAMFERA": 0,
        "$CHAMFERB": 0,
        "$CHAMFERC": 0,
        "$CHAMFERD": 0,
        "$SKPOLY": 0,
        "$TDCREATE": 2455379.770598715,
        "$TDUCREATE": 2455379.770598715,
        "$TDUPDATE": 2455614.973897338,
        "$TDUUPDATE": 2455614.973897338,
        "$TDINDWG": 1.6838402431,
        "$TDUSRTIMER": 1.6838402431,
        "$USRTIMER": 1,
        "$ANGBASE": 0,
        "$ANGDIR": 0,
        "$PDMODE": 0,
        "$PDSIZE": 0,
        "$PLINEWID": 0,
        "$SPLFRAME": 0,
        "$SPLINETYPE": 6,
        "$SPLINESEGS": 8,
        "$HANDSEED": "BFA8",
        "$SURFTAB1": 6,
        "$SURFTAB2": 6,
        "$SURFTYPE": 6,
        "$SURFU": 6,
        "$SURFV": 6,
        "$UCSBASE": "",
        "$UCSNAME": "",
        "$UCSORG": {
            "x": 0,
            "y": 0,
            "z": 0
        },
        "$UCSXDIR": {
            "x": 1,
            "y": 0,
            "z": 0
        },
        "$UCSYDIR": {
            "x": 0,
            "y": 1,
            "z": 0
        },
        "$UCSORTHOREF": "",
        "$UCSORTHOVIEW": 0,
        "$UCSORGTOP": {
            "x": 0,
            "y": 0,
            "z": 0
        },
        "$UCSORGBOTTOM": {
            "x": 0,
            "y": 0,
            "z": 0
        },
        "$UCSORGLEFT": {
            "x": 0,
            "y": 0,
            "z": 0
        },
        "$UCSORGRIGHT": {
            "x": 0,
            "y": 0,
            "z": 0
        },
        "$UCSORGFRONT": {
            "x": 0,
            "y": 0,
            "z": 0
        },
        "$UCSORGBACK": {
            "x": 0,
            "y": 0,
            "z": 0
        },
        "$PUCSBASE": "",
        "$PUCSNAME": "",
        "$PUCSORG": {
            "x": 0,
            "y": 0,
            "z": 0
        },
        "$PUCSXDIR": {
            "x": 1,
            "y": 0,
            "z": 0
        },
        "$PUCSYDIR": {
            "x": 0,
            "y": 1,
            "z": 0
        },
        "$PUCSORTHOREF": "",
        "$PUCSORTHOVIEW": 0,
        "$PUCSORGTOP": {
            "x": 0,
            "y": 0,
            "z": 0
        },
        "$PUCSORGBOTTOM": {
            "x": 0,
            "y": 0,
            "z": 0
        },
        "$PUCSORGLEFT": {
            "x": 0,
            "y": 0,
            "z": 0
        },
        "$PUCSORGRIGHT": {
            "x": 0,
            "y": 0,
            "z": 0
        },
        "$PUCSORGFRONT": {
            "x": 0,
            "y": 0,
            "z": 0
        },
        "$PUCSORGBACK": {
            "x": 0,
            "y": 0,
            "z": 0
        },
        "$USERI1": 0,
        "$USERI2": 0,
        "$USERI3": 0,
        "$USERI4": 0,
        "$USERI5": 0,
        "$USERR1": 0,
        "$USERR2": 0,
        "$USERR3": 0,
        "$USERR4": 0,
        "$USERR5": 0,
        "$WORLDVIEW": 1,
        "$SHADEDGE": 3,
        "$SHADEDIF": 70,
        "$TILEMODE": 0,
        "$MAXACTVP": 64,
        "$PINSBASE": {
            "x": 0,
            "y": 0,
            "z": 0
        },
        "$PLIMCHECK": 0,
        "$PEXTMIN": {
            "x": -0.25,
            "y": -0.1305130401371635,
            "z": 0
        },
        "$PEXTMAX": {
            "x": 16.5,
            "y": 10.61948695986283,
            "z": 0
        },
        "$PLIMMIN": {
            "x": -0.2499321702471884,
            "y": -0.3750450970290743
        },
        "$PLIMMAX": {
            "x": 16.75006734916103,
            "y": 10.62495466267503
        },
        "$UNITMODE": 0,
        "$VISRETAIN": 1,
        "$PLINEGEN": 0,
        "$PSLTSCALE": 1,
        "$TREEDEPTH": 3020,
        "$CMLSTYLE": "Standard",
        "$CMLJUST": 0,
        "$CMLSCALE": 1,
        "$PROXYGRAPHICS": 1,
        "$MEASUREMENT": 0,
        "$CELWEIGHT": -1,
        "$ENDCAPS": 0,
        "$JOINSTYLE": 0,
        "$LWDISPLAY": false,
        "$INSUNITS": 0,
        "$HYPERLINKBASE": "",
        "$STYLESHEET": "Florida.ctb",
        "$XEDIT": true,
        "$CEPSNTYPE": 0,
        "$PSTYLEMODE": true,
        "$FINGERPRINTGUID": "{E451773B-08B1-4376-9DA5-8F5F0D6C19EA}",
        "$VERSIONGUID": "{6EFBAB5D-9128-4E51-B2D4-822E03762ACA}",
        "$EXTNAMES": true,
        "$PSVPSCALE": 0,
        "$OLESTARTUP": false,
        "$SORTENTS": 127,
        "$INDEXCTL": 0,
        "$HIDETEXT": 1,
        "$XCLIPFRAME": 0,
        "$HALOGAP": 0,
        "$OBSCOLOR": 257,
        "$OBSLTYPE": 0,
        "$INTERSECTIONDISPLAY": 0,
        "$INTERSECTIONCOLOR": 257,
        "$DIMASSOC": 2,
        "$PROJECTNAME": "",
        "$CAMERADISPLAY": false,
        "$LENSLENGTH": 50,
        "$CAMERAHEIGHT": 0,
        "$STEPSPERSEC": 2,
        "$STEPSIZE": 6,
        "$3DDWFPREC": 2,
        "$PSOLWIDTH": 0.25,
        "$PSOLHEIGHT": 4,
        "$LOFTANG1": 1.570796326794896,
        "$LOFTANG2": 1.570796326794896,
        "$LOFTMAG1": 0,
        "$LOFTMAG2": 0,
        "$LOFTPARAM": 7,
        "$LOFTNORMALS": 1,
        "$LATITUDE": 37.795,
        "$LONGITUDE": -122.394,
        "$NORTHDIRECTION": 0,
        "$TIMEZONE": -8000,
        "$LIGHTGLYPHDISPLAY": 1,
        "$TILEMODELIGHTSYNCH": 1,
        "$CMATERIAL": "96",
        "$SOLIDHIST": 0,
        "$SHOWHIST": 1,
        "$DWFFRAME": 2,
        "$DGNFRAME": 0,
        "$REALWORLDSCALE": true,
        "$INTERFERECOLOR": 1,
        "$INTERFEREOBJVS": "A3",
        "$INTERFEREVPVS": "BF84",
        "$CSHADOW": 0,
        "$SHADOWPLANELOCATION": 0
    }
}